junit5-mockito-extension/build.gradle

```diff
@@ -40,7 +40,7 @@ junit5 {
 
 dependencies {
 	// Needed as compile since we're building a JUnit 5 extension here
-	compile 'org.mockito:mockito-core:1.+'
+	compile 'org.mockito:mockito-core:2.0.69-beta'
```

Comment:
I tried that as both 2+ and 2.+ and it didn't work - I don't know Gradle well but assumed it was due to the -beta label.